packages/inject/src/themes/shareable.css

```diff
@@ -1206,6 +1206,6 @@ Footer
   }
 }
 
-#react-select-2-listbox {
```

Comment:
**note**: This was only working for the first dropdown in the chatbot, as the number in the middle (the `2`) is incremented for each dropdown that appears.